test/pooling2d.hpp

```diff
@@ -2,7 +2,7 @@
  *
  * MIT License
  *
- * Copyright (c) 2019 Advanced Micro Devices, Inc.
+ * Copyright (c) 2024 Advanced Micro Devices, Inc.
```

Comment:
I dont think this is required, since you are not touching the rest of the file.

Reply:
There are a few minor lines added to the end of this file in a previous PR for gtest support. I'm not certain it's enough to change the date here.

Reply:
reverted